pandas/tests/arrays/integer/test_function.py

```diff
@@ -113,6 +113,17 @@ def test_value_counts_empty():
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize("skipna", [True, False])
+@pytest.mark.parametrize("min_count", [0, 4])
+def test_integer_array_sum(skipna, min_count):
```

Comment:
is e.g. Series[Int64].sum() or DataFrame[Int64].sum() fixed by this?

Reply:
Those are actually already working, just not for IntegerArray specifically